Simplenote/HuggableTableView.swift

```diff
@@ -5,6 +5,20 @@ import UIKit
 //
 class HuggableTableView: UITableView {
 
+    lazy var maxHeightConstraint: NSLayoutConstraint = {
+        let constraint = heightAnchor.constraint(lessThanOrEqualToConstant: CGFloat.greatestFiniteMagnitude)
+        constraint.priority = UILayoutPriority(rawValue: 999)
+        constraint.isActive = true
+        return constraint
+    }()
+
+    lazy var minHeightConstraint: NSLayoutConstraint = {
+        let constraint = heightAnchor.constraint(greaterThanOrEqualToConstant: 0)
+        constraint.priority = UILayoutPriority(rawValue: 999)
+        constraint.isActive = true
+        return constraint
+    }()
+
```

Comment:
TBH I'm not 100% sure this is the **right** place for these constraints (specially because they're not initialized right here).

But considering that it's analog to `heightAnchor`, so to speak, sounds reasonable?

Reply:
I thought to keep them lazy because they're not always needed, but if a user of this class wants to set min / max height, they provide an easy way to do it.